man/systemd-id128.xml

```diff
@@ -165,6 +169,29 @@ NAME      ID
 user-home 773f91ef66d449b5bd83d683bf40ad16
       </programlisting>
     </example>
+
+    <example>
+      <title>Generate an application-specific UUID</title>
+
+      <programlisting>
+$ systemd-id128 machine-id -u
+3a9d668b-4db7-4939-8a4a-5e78a03bffb7
+
+$ systemd-id128 new -u
+1fb8f24b-02df-458d-9659-cc8ace68e28a
+
+$ systemd-id128 --app=1fb8f24b-02df-458d-9659-cc8ace68e28a
```

Comment:
this does not actually work:

```
Command verb required.
```

And I don't grok what it even was supposed to do?

Reply:
→ https://github.com/systemd/systemd/pull/29098